### DISCUSSION TITLE
A slight improvement of a quickstart example

### DISCUSSION THREAD

**Discussion by QuarticCat** [sayanarijit/xplr#254] [Show and tell]
Try this:

```bash
# Change directory
alias xcd='cd "${$(xplr):-.}"'
```

If we decide not to navigate, xplr will return nothing, resulting in a bare `cd`, which is equivalent to `cd ~`. After changing to `"${$(xplr):-.}"`, that situation will result in a `cd .`, which keeps still. This grammar is available in bash and zsh.

Also, in **zsh**, we can implement fish-like navigation via xplr by this line:

```zsh
# [Ctrl-N] run xplr to navigate
bindkey -s '^N' '^Qcd "${$(xplr):-.}"\n'
```

where `^Q` means push-line, and `\n`/`^J` means accept-line.

If we don't want to add this line into history, we can firstly open `hist_ignore_space` by

```zsh
setopt hist_ignore_space
```

Then change the above line into

```zsh
bindkey -s '^N' '^Q cd "${$(xplr):-.}"\n'
```

Note the space before `cd`.

[My dotfiles](https://github.com/QuarticCat/dotfiles/blob/main/dot_zshrc)

**Comment by sayanarijit** [comment]
Hey, awesome tips. I'm afraid these syntaxes might confuse the newcomers. So I'll link it in as "Advanced Tips & Aliases" instead.

Also, I think we can combine this with the [tips shared here](https://github.com/sayanarijit/xplr/discussions/241#discussioncomment-832755).
